### PR TITLE
[15.0][MIG] animal: Migration to 15.0

### PR DESCRIPTION
**Pull request by moaadbourhim**
Migration of animal to 15.0

### CODE REVIEW

**Review by max3903 — APPROVED** [review]

File: animal/tests/test_animal.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+from odoo.tests.common import TransactionCase
+
+
+# @tagged('-at_install', 'post_install')
+class TestAnnimalState(TransactionCase):
```
Comment (max3903) [inline animal/tests/test_animal.py]:
```suggestion
class TestAnimalState(TransactionCase):
```

File: animal/tests/test_animal.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    def setUp(self, *args, **kwargs):
+        super(TestAnnimalState, self).setUp(*args, **kwargs)
```
Comment (max3903) [inline animal/tests/test_animal.py]:
```suggestion
        super(TestAnimalState, self).setUp(*args, **kwargs)
```

**Comment by max3903** [comment]
@sbidoul Can you help us here with the Runbot error:
```
Unable to find image 'oca-partner-contact:563c219_env_1_job_1' locally
docker: Error response from daemon: pull access denied for oca-partner-contact, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.
```

**Comment by sbidoul** [comment]
@max3903 runbot is not supported for 15.0 branches yet.

**Pull request merged**